#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enable_flag", bool_t,    0, "the enable flag to config", False)
gen.add("time_diff", double_t,    0, "the diffenrence of the time stamp", 0.0, -0.5, 0.5)

exit(gen.generate(PACKAGE, "jsk_perception", "TimeStamp"))
